docs/api.rst

```diff
@@ -12459,3 +12459,37 @@ Getting all available counterparties
 
   :statuscode 200: Information was correctly generated
   :statuscode 500: Internal rotki error
+
+Getting valid locations for history events
```

Comment:
What is this endpoint? What do you mean by valid location for history events? There is no such thing. Please **ask me** or read the code before proceeding to make PRs.

I already told you in our call all locations are valid. There already is events from a lot of different locations imported by the generic CSV importer. We discussed this.